src/symbolic/symbolic_wasm_ffi.ml

```diff
@@ -24,13 +24,15 @@ module M :
   let assert' (i : Value.int32) : unit Choice.t =
     Choice.assertion @@ Value.I32.to_bool i
 
+  let symbol_bool () =
+    Choice.with_new_symbol (Ty_bitv 1) (fun sym ->
+        Expr.cvtop (Ty_bitv 32) (Smtml.Ty.Zero_extend 31) (Expr.symbol sym) )
```

Comment:
```suggestion
        Expr.cvtop (Ty_bitv 32) (Zero_extend 31) (Expr.symbol sym) )
```

Also I'm a bit worried about creating a symbolic bitvector of size 1, I don't know if it will play nice with the current model generation in smtml. Do we have up to date results of test-comp? I can leave it running over night if need be to verify my concern

Reply:
> Also I'm a bit worried about creating a symbolic bitvector of size 1, I don't know if it will play nice with the current model generation in smtml. Do we have up to date results of test-comp? I can leave it running over night if need be to verify my concern (but don't let this delay this PR, it's something I can address in the upcoming release of smtml)

Apparently I already did this in `smtml.0.2.2` so we should be good!